internal/campaign/campaign.go

```diff
@@ -60,10 +68,13 @@ func OpenCampaign() (*Campaign, error) {
 	}, nil
 }
 
-func readReposTxtFile() ([]Repo, error) {
-	file, err := os.Open("repos.txt")
+func readReposTxtFile(filename string) ([]Repo, error) {
+	if filename == "" {
+		return nil, errors.New("no repos filename to open")
+	}
+	file, err := os.Open(filename)
 	if err != nil {
-		return nil, errors.New("unable to open repos.txt file")
+		return nil, fmt.Errorf("unable to open %s file", filename)
```

Comment:
```suggestion
		return nil, fmt.Errorf("unable to open repos file: %s", filename)
```
Just so the user knows why the program is trying to open the named file